dojo/utils.py

```diff
@@ -2633,10 +2633,8 @@ def get_open_findings_burndown(product):
                         info_count -= 1
 
         f_day = [critical_count, high_count, medium_count, low_count, info_count]
-        if min(f_day) < running_min:
-            running_min = min(f_day)
-        if max(f_day) > running_max:
-            running_max = max(f_day)
+        running_min = min(running_min, f_day)
+        running_max = max(running_max, f_day)
```

Comment:
You are comparing a list and a value. If you would like to find `min` in "value and all elements in list", the list needs to be expanded.

```python
>>> a = [1, 2]
>>> b = 3
>>> min(a, b)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'int' and 'list'
>>> min(*a, b)
1
```

Reply:
This hint resolved some errors, thank you :-)